src/Umbraco.Infrastructure/Migrations/Upgrade/V_12_0_0/UseNvarcharInsteadOfNText.cs

```diff
@@ -40,6 +50,13 @@ private void MigrateNtextColumn<TDto>(string columnName, string tableName, Expre
             return;
         }
 
+        // since it's ntext to nvarchar(max) we should be able to just raw query this without having issues with fallback values on sql server
+        if (Database.DatabaseType != DatabaseType.SQLite && Database.DatabaseType != DatabaseType.SQLCe)
```

Comment:
Maybe not test for SQL CE here? It obviously doesn't harm, but CE hasn't been supported since V9, so to avoid any future confusion, maybe just remove this test?